src/popup/App.vue

```diff
@@ -218,6 +222,15 @@ export default defineComponent({
       }
     });
 
+    // Redirect to account page if no multisig accounts are present
+    // only if current page is a multisig page
+    watch(multisigAccounts,
+      (value) => {
+        if (!value?.length && route.meta.isMultisig) {
+          router.replace({ name: ROUTE_ACCOUNT });
+        }
+      });
```

Comment:
Something is messed up with the indentation here. As always I suggest to take one approach: all arguments in separate lines or in one line. Never mixed.